BUILD

```diff
@@ -51,3 +51,34 @@ container_test(
     configs = ["//tests/container/configs:test.yaml"],
     image = "//testdata:link_with_files_base",
 )
+
+filegroup(
```

Comment:
Add a TODO to remove this once project can be copied otherwise

Reply:
Done